README.md

```diff
@@ -2,179 +2,112 @@
+note: some options have default so they are enabled even if you don't specify them.
 
-```Shell
-pushd C:\Program Files\nodejs && node.exe C:\Users\yourwindowsusername\AppData\Roaming\npm\node_modules\gtp2ogs\gtp2ogs.js --username <yourbotusername> --apikey <apikey> <gtp2ogsargument1> <gtp2ogsargument2> -- C:\Users\path\to\your\ai\executable.exe <botargument1> <botargument2>
-```
+After that, specify your bot command (and its arguments) using a , add a ` -- `
+ (with one space before and one space after the `--` ), to separate `options`
+ from your bot path and `bot arguments`.
 
-note: for all operating systems, in `--`, the spaces after `<gtp2ogsarguments>`
-and before `/path/to/your/bot.executable` are important: they separate gtp2ogs
-arguments from your bot arguments
+All in all, general use is something like this:
 
-note 2: the number of `<gtp2ogsarguments>` and `<botarguments>` is not limited,
-here only 2 were shown but it possible to use for example 3,4,5 , or as many as
-you want
-  
-note 3: to play on [beta OGS server](https://beta.online-go.com/) instead of the
-[OGS server](https://online-go.com/), add the `--beta` argument
+`path/to/node_executable path/to/gtp2ogs.js --username <yourbotusername> --apikey <apikey> --option1 <argument1> --option2 <argument2> -- path/to/your/ai/executable <botoption1> <botargument1> <botoption2> <botargument2>`
 
-## Options
+## Example of use
 
-You can find a list of all possible to use gtp2ogs arguments [here](/docs/OPTIONS-LIST.md).
+In this example AI is PhoenixGo, but it works in a similar way for any AI.
 
-Before putting `<gtp2ogsarguments>`, you have to put the options [username](https://github.com/online-go/gtp2ogs/blob/devel/docs/OPTIONS-LIST.md#username)
-and [apikey](https://github.com/online-go/gtp2ogs/blob/devel/docs/OPTIONS-LIST.md#apikey)
-first.
+The number of gtp2ogs options and of bot arguments is free.
 
-Then, choose the wanted `<gtp2ogsarguments>` from the above [list of all options](/docs/OPTIONS-LIST.md) section (no need to use all these options, only the ones you want),
-and separate each option with a space
+### For Windows
 
-After that, add a `--` (with one space before and one space after the `--` ),
-to separate `<gtp2ogsarguments>` from your bot path and `<botarguments>`, as
-shown in
-[Most common usage earlier](#4-most-common-usage--start-gtp2ogsjs-using-nodejs)
+`C:\Program Files\nodejs\node.exe C:\Users\yourusername\gtp2ogs-node\node_modules\gtp2ogs\gtp2ogs.js --apikey 5rg46ze84f68e4g6v4e4g68es4g8 --username somebot --debug --ogspv PhoenixGo --noclock --unrankedonly --minrank 1d --noautohandicapranked --maxhandicap 0 --boardsizes 19 --komis 7.5 --speeds live --maxmaintimelive 600 --maxperiodsliveranked 5 --maxperiodsliveunranked 10 --maxperiodtimeliveranked 60 --maxperiodtimeliveunranked 180 -- C:\Users\yourusername\PhoenixGo\mcts\mcts_main --gtp --config_path C:\Users\yourusername\PhoenixGo\etc\mcts_1gpu_beta.conf --logtostderr --v 1`
```

Comment:
No need to use `C:\Program Files\nodejs\` if you installed node.js and have it in your path.

Reply:
then just node.exe ?